app/views/contact/govuk/accessible_format_requests/start_page.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title do %>
-  <%= t("controllers.contact.govuk.accessible_format_requests.start_page.request_heading") %>
+  <%= t("controllers.contact.govuk.accessible_format_requests.start_page.request_heading", attachment_title: attachment_title) %>
```

Comment:
This translation won't need the variable passed through

Reply:
Thanks Ed, well spotted!